CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Prowide Core - CHANGELOG
 
+#### 10.3.0 - SNAPSHOT
+  * SWIFT Standard release update 2025 (live 22 November 2025)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)